src/test/java/com/assignmentTest/Assignment1Test.java

```diff
@@ -0,0 +1,79 @@
+package com.assignmentTest;
+
+import java.lang.*;
+
+import com.assignment.Assignment1;
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.Test;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+public class Assignment1Test {
+    private Logger logger = LoggerFactory.getLogger(this.getClass());
+
+    @Test
+    public void testCovertDataType() {
+        Assignment1 assignment1 = new Assignment1();
+        Object object = null;
```

Comment:
Why are we using an object, why not use primitive data types?

Reply:
getClass() doesnt work on that. So for assertion I used this

Reply:
@dheerajsrinath Can we try this?